DESCRIPTION

```diff
@@ -1,25 +1,50 @@
 Package: tractr
-Title: Generalized Additive Models for analyzing diffusion MRI tract profiles
+Title: Analyzing diffusion MRI tract profiles
 Version: 0.0.0.9000
-Authors@R: 
-    person(given = "Adam",
+Authors@R:
+    c(person(given = "Adam",
            family = "Richie-Halford",
            email = "richiehalford@gmail.com",
            role = c("aut", "cre"),
-           comment = c(ORCID = "0000-0001-9276-9084"))
-Description: A toolbox for analyzing diffusion MRI tract profiles using
-    generalized additive models.
+           comment = c(ORCID = "0000-0001-9276-9084")),
+    person(given = "Kelly",
+           family = "Chang",
+           email = "XXX",
+           role = c("aut"),
+           comment = c(ORCID = "XXX")),
+    person(given = "Teresa",
+           family = "Gomez",
+           email = "XXX",
+           role = c("aut"),
+           comment = c(ORCID = "XXX")),
+    person(given = "McKenzie",
+           family = "Hagen",
+           email = "XXX",
+           role = c("aut"),
+           comment = c(ORCID = "XXX")),
```

Comment:
@mckenziephagen : what email would you like to use here, and what's your ORCID?

Reply:
ORCID: 0000-0002-7454-8189
email: mphagen@uw.edu